interactive-calculator/src/containers/ItemsForm/ItemsForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+export default ({ setItems} ) => {
+
+  const [item, setItem] = useState({
+    concept: '',
+    price: 0,
+    priority: 1, 
+   })
+
+  return (
+    <form onSubmit={(e) => {
+        e.preventDefault();
+        setItems((prevItems) => [...prevItems, item])
+      }}>
+      <h2>Add Item</h2>
+      <label htmlFor='concept'>Concept</label>
+      <input 
+        name="concept"
+        type="text"
+        value={item.concept}
+        onChange={(e) => {
```

Comment:
Maybe try converting this anonymous function you call on the onChange to be a named function called handleChange()?